packages/saved-views-react/src/translation/displayStyleExtractor.ts

```diff
@@ -741,6 +741,13 @@ export const extractDisplayStyle = (data: object, viewState?: ViewState) => {
   if ("displayStyleProps" in data) {
     styles = (data as LegacySavedView2d).displayStyleProps.jsonProperties?.styles;
     applyExtraction(styles, output, displayStylesLegacyMapping);
+    if (output?.thematic?.range) {
+      output.thematic.range =
+        Array.isArray(output.thematic.range) &&
+        output.thematic.range.length === 0
+          ? undefined
```

Comment:
Why not delete the property instead of setting to undefined, like in utils?

Reply:
Ah, you're right. I didn't change this saved-views implementation when I changed the utils one. Fixing now...